packages/dcos-integration-test/extra/conftest.py

```diff
@@ -21,6 +21,31 @@
 LOGIN_PW = os.getenv('DCOS_LOGIN_PW')
 
 
+def pytest_configure(config):
+    config.addinivalue_line(
+        'markers', 'order(n): mark test to run in Nth order.'
+        'Negative numbers represent position from the end.'
+        'EG: 1, 2, 3, ... unordered ... -3, -2, -1')
+
+
+def pytest_collection_modifyitems(session, config, items):
+    """Reorders test using order mark
+    """
+    first = []
+    unordered = []
+    last = []
+    for item in items:
+        if not hasattr(item.obj, 'order'):
```

Comment:
This implementation doesnt account for negatives (index from end) and it will put unordered tests first. Once you try to do those two cases, I think we end up with what is already here